src/Symfony/Component/Uid/DsHashablePolyfill.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Uid;
+
+interface DsHashablePolyfill
```

Comment:
Not sure about the whole polypill things here, but wouldn't it make sense to move this to the polypill package?

Reply:
There is a polyfill package already: https://github.com/php-ds/polyfill

Reply:
An approach that could work would be to declare an internal HashableInterface who's definition would depend on the existence of the Ds one.

Reply:
so how about this?